### ISSUE TITLE
Revisit Context Selector placement

### ISSUE THREAD

**Issue by mcarrano** [patternfly/patternfly-design-kit#184]
Multiple issues/requests have surfaced concerning the Context Selector. These mainly focus on placement of the control

These issues are being tracked in a Google Doc here: https://docs.google.com/document/d/1c0tKt8DBNWx67QmLocvpUnbR_EeJodxKSyaHzLVXxIw/edit?usp=sharing

A successful conclusion to this story will include one or more specific enhancements that can be implemented in PatternFly.

cc: @rachael-phillips @LHinson @catrobson

**Comment by ajacobs21e** [comment]
@mcarrano is there a disabled state for the context selector?

**Comment by mcarrano** [comment]
@ajacobs21e It does not look like we ever defined one and not sure if it is needed. In theory, you can use any type of Select component to serve as a context selector, so I guess we would just pick up the states defined for those components.

**Comment by ajacobs21e** [comment]
[Context Selector.sketch.zip](https://github.com/patternfly/patternfly-design-kit/files/4488901/Context.Selector.sketch.zip)
@gdoyle1 I did my best guess for where these components will live (how to name the symbol) - please check these and edit as necessary.
I left in all states just in case, delete as necessary. Thanks!

@mcarrano at least for the 'in nav' version, the select is a different color than the existing dark theme dropdown. We should consider updating the dark theme dropdowns to this style (basically a #151515 background instead of #393F44) once we look at dark theme more in depth.

**Comment by mcarrano** [comment]
Yeah, I was thinking we might run up into some of these issues. I believe we have an issue in the backlog to build a core demo of this which is where these things will surface. It may be that we need a unique styling for a Select inside of the nav.

cc @mceledonia

**Comment by ajacobs21e** [comment]
@mcarrano is it premature to add this to the sketch library? in case things might change during core development?

**Comment by mcarrano** [comment]
@ajacobs21e I kind of think it is. All of this is closely tied to the masthead refactoring work that has lots of moving parts. So I would be OK to hold off.

**Comment by gdoyle1** [comment]
@mcarrano @ajacobs21e Ok I'll put this on hold for now but then hopefully we can use your work whenever we add this to the Sketch library, Allie! Thank you for taking the time on it :)

**Comment by yih-wang** [comment]
Hi, is there any plan to release this component? The Keycloak development is about to start in June/July and this context selector is in our first delivery package. So I want to check the status.

**Comment by mcarrano** [comment]
@yih-wang can you elaborate more on exactly what functionality you need? If you need to place a context selector in the masthead, that will require updates to the masthead that are scheduled in our 2020.08 milestone at the end on June.

**Comment by yih-wang** [comment]
@mcarrano Yes, that's exactly what I need. Does it mean the code will be available at the end of June?

**Comment by mcarrano** [comment]
@yih-wang That is when our HTML/CSS is scheduled to be completed. There is an open issue here: https://github.com/patternfly/patternfly/issues/2800. Is KeyCloak using PF React? I would expect the React effort to lag by another 3 weeks which will place us into July. I am copying in @LHinson so she is aware of this timeline.

**Comment by yih-wang** [comment]
@mcarrano Yes, we will use the React library. Thanks for the information!
